api/proto/auth/auth.proto

```diff
@@ -0,0 +1,21 @@
+// Copyright(c) 2020 Zededa, Inc.
+// All rights reserved.
+
+syntax = "proto3";
+
+option go_package  = "github.com/lf-edge/eve/api/go/auth";
+option java_package = "com.zededa.cloud.uservice.proto";
+
+
+message AuthBody {
+    bytes  payload = 1;
+}
+
+message AuthContainer {
+    AuthBody authPayload = 1;      // envolope body, a marshalled protobuf data or it can be null
+    // if the length of senderCertHash received is not 16 bytes, then the protobuf
+    // message either is not AuthContainer type, or is corrupted. Otherwise, the
+    // reciever may not have the sender's signing certificate
+    string   senderCertHash = 2;   // 16 bytes in length, 1st 16 char of sender siging cert sha256 hash
```

Comment:
We should check this works for the controller. When we talked we said we'd add the device UUID for the device to controller direction, but if a short hash of the cert works in both directions that is more symmetric.

I think we also optionally need the full sender certificate to be able to do the register API, but perhaps the short hash of the onboarding cert is sufficient for the controller to find the onboarding cert.

So let's merge this will we double-check on the controller side.